firestore.rules

```diff
@@ -1,12 +1,19 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /config/{document=**} {
+    match /config/{configFile} {
         allow write: if request.auth.uid != null;
         allow read: if true;
     }
-    match /{document=**} {
+    match /users/{userId} {
         allow write, read: if request.auth.uid != null;
     }
+    match /chatsv2/{chatId} {
+    		allow write: if request.auth.uid != null && request.resource.data.senderId == request.auth.uid;
+        allow read: if request.auth.uid != null && (resource.data.recipientId == null || resource.data.recipientId == request.auth.uid);
```

Comment:
Does this allow you to read to messages you sent (where you are senderId)?

Reply:
that's true there 's a mistake